myria3d/models/modules/pointnet.py

```diff
@@ -0,0 +1,126 @@
+import torch
```

Comment:
This needs testing, tests can be added in https://github.com/IGNF/myria3d/blob/main/tests/myria3d/models/modules/ (see https://github.com/IGNF/myria3d/blob/main/tests/myria3d/models/modules/test_randla_nets.py for an example)